x/launch/client/cli/tx_settle_request.go

```diff
@@ -0,0 +1,50 @@
+package cli
+
+import (
+	"strconv"
+
+	"github.com/cosmos/cosmos-sdk/client"
+	"github.com/cosmos/cosmos-sdk/client/flags"
+	"github.com/cosmos/cosmos-sdk/client/tx"
+	"github.com/spf13/cobra"
+	"github.com/tendermint/spn/x/launch/types"
+)
+
+func CmdSettleRequest() *cobra.Command {
+	cmd := &cobra.Command{
+		Use:   "settle-request [approve/reject] [chainID] [requestID]",
```

Comment:
```suggestion
		Use:   "settle-request [approve|reject] [chainID] [requestID]",
```

I would prefer using `|` that means or